### ISSUE TITLE
Pedestals are duplicating entities

### ISSUE THREAD

**Issue by fozolo** [minetest-mods/homedecor_modpack#405]
For years I've been getting errors like this:

> 2018-03-10 22:00:27: WARNING[Server]: ServerEnv: Trying to store id = 34946 statically but block (72,1,-40) already contains 512 objects. Forcing delete.

I've narrowed it down to `itemframes:pedestal`.  Every time I enter the area of the map with a pedestal, it adds a new entity until eventually there are too many and then leaving the map area produces the warning above.  I have been unable to reproduce this on my dev map, but I made a copy of the Fozland map and can reproduce this reliably on that map.

I've been staring at the code for awhile but I can't see any reason for this.

**Comment by sofar** [comment]
tested using 4d9c4c2410cac3d7f3ab83d66121f830f826cc0c but I can't reproduce. I annotated the code to show me a line for each activated (loaded from mapblock storage) and it only ever shows me the 2 testing pedestals I made, but never more items if I leave and come back in the zone again.

**Comment by fozolo** [comment]
After more testing, I found this is happening to some itemframes as well.  I have also confirmed my suspicion that this is a race condition bug.  After adding a lot of debugging print outs, I've found that the  ABM runs on the effected itemframes and pedestals before their respective entities have been loaded.  This causes the abm to find zero entities which results in it adding a new one.  I tried replacing the ABM with an LBM but it has the same problem.
